nightlies/ansible-crons/crons.yaml

```diff
@@ -0,0 +1,53 @@
+# *** SEPIA LAB ***
+- hosts: teuthology.front.sepia.ceph.com
+  vars:
+    qa_email: "-e ceph-qa@ceph.com"
+    cron_wrapper: "/home/teuthology/bin/cron_wrapper"
+    sepia_cron_user: "teuthology"
+    rados_command: "/home/teuthology/bin/schedule_rados.sh"
+
+    suites:
+
```

Comment:
I'd get rid of this spacing so it's clear `suites` and its items, `rgw`, `fs`, and `rados` are items.

    #Example:
      suites:
        rgw:
          name: rgw
          etc.
        fs:
          name: fs
          etc.